components/active_filters.py

```diff
@@ -8,17 +8,17 @@ class ActiveFilters:
     @staticmethod
     def get_component():
         widget = dbc.Col(
+            width={"width": "60%"},
```

Comment:
Can be replaced with `className="col-6"` or another bootstrap class like that :)

Reply:
Still ^